Spock-core/src/Web/Spock/Internal/Wire.hs

```diff
@@ -192,6 +193,18 @@ newtype ActionCtxT ctx m a
 instance MonadTrans (ActionCtxT ctx) where
     lift = ActionCtxT . lift . lift
 
+data SpockConfigInternal
+    = SpockConfigInternal
+    { sci_maxRequestSize :: Maybe Word64
+    , sci_errorHandler :: Status -> IO Wai.Application
+    }
+
+defaultSpockConfigInternal :: SpockConfigInternal
+defaultSpockConfigInternal = SpockConfigInternal Nothing defaultErrorHandler
+  where
+    defaultErrorHandler status = return $ \_ respond ->
+      respond $ Wai.responseBuilder status [] ""
```

Comment:
We should probably put something more informative here, because this only happens if the user defined error handler fails with an exception